src/page/menu.rs

```diff
@@ -136,49 +144,126 @@ fn NsfwToggle() -> impl IntoView {
 #[component]
 pub fn Menu() -> impl IntoView {
     let (is_connected, _) = account_connected_reader();
+    let query_map = use_query_map();
+    let show_content_modal = create_rw_signal(false);
+
+    let can_res = authenticated_canisters();
+    let is_authorized = create_local_resource_with_initial_value(
+        move || MockPartialEq(can_res()),
+        move |can_res| async move {
+            let canisters = can_res.0?.ok()?;
+            canisters.is_user_authorized_to_seed_content().await.ok()
+        },
+        Some(Some(false)),
+    );
```

Comment:
Probably want to cache this value in context (like how we cache posts)

Reply:
Done